README.md

```diff
@@ -80,6 +80,11 @@ _$5K to $50K for open source projects by independent researchers and developers.
 https://www.openhumans.org/grants/ <br>
 _$5,000 for human data projects (ex. genetic, activity or social media data) that help grow the Open Humans ecosystem._
 
+## OffLocalhost
+https://www.offlocalhost.xyz/ <br>
+Instant funding to kick-start your ideas. A fellowship, guiding your every step. <br>
```

Comment:
better mention clearly that is for startups... (is it even microgrant?)

Reply:
hey sure made a commit. the check sizes usually vary as their website says as low as a a few hundred dollars to tens of thousands. thanks

Reply:
I am rather thinking about this being VC funding, not a grant - not funding size.

Reply:
Ahh I see what you mean. They do have equity-free funding options too. Quoting their site below. Although if it's still an issue, I'll close the pr. 

> The choice is entirely yours. As a fellowship member, you have the option of light sponsorship funding and various support forms without any equity exchange. Alternatively, you could choose an equity deal for more substantial support and funding, typically between $10,000 and $50,000 for a 3% to 10% equity stake. 
(OffLocalhost website) 

Best,